zititest/zitilab/chaos/chaos.go

```diff
@@ -92,4 +122,8 @@ func ValidateUp(run model.Run, spec string, concurrency int, timeout time.Durati
 			time.Sleep(time.Second)
 		}
 	})
+	if err == nil {
+		pfxlog.Logger().Infof("all %v components for spec '%s' are running", len(components), spec)
```

Comment:
I still dislike %v due to its lack of complaints when refactoring. In this scenario %d would work just fine.

Not demanding a change, just venting.